config/config.example.xml

```diff
@@ -42,26 +42,30 @@
                 <soffice>/usr/bin/soffice</soffice>
             </config>
         </pdfshadow>
-    
+
         <elasticsearch enabled="1" class="\Balloon\App\Elasticsearch">
             <config>
                 <hosts>
                     <server>http://localhost:9200</server>
                 </hosts>
-                <index>balloon</index> 
+                <index>balloon</index>
             </config>
         </elasticsearch>
 
+        <ClamAv enabled="0" class="\Balloon\App\ClamAv">
+            <config/>
```

Comment:
example config should declare all possible configurations socket/max_stream_size

Reply:
fixed in #70b21c2